members.md

```diff
@@ -92,6 +89,9 @@ members:
       - name: Master's students (Reseach Master's)
         full: False
         list:
+          - name: Sarthak Shetty (MSME) -> Path Robotics
```

Comment:
Can you isolate just this change to members.md as a PR?

Reply:
Thank you for the feedback. I've rolled back the changes to `main.css` and `area.html`.